src/theme.css

```
@@ -0,0 +1,101 @@
+}
+
+ion-header ion-title {
+  font-weight: 800;
+  font-size: 18px;
+  letter-spacing: -0.02em;
+  color: var(--brand-primary);
+}
+
+ion-header ion-buttons ion-button {
+  --color: var(--text-secondary);
+  font-weight: 600;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)